src/index.js

```diff
@@ -1,15 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
```

Comment:
There is another issue in your code with the import statement for ReactDOM. You are trying to import ReactDOM from react-dom/client, but ReactDOM should be imported from 'react-dom'.  Here's the corrected import statement:
`import ReactDOM from 'react-dom';`

Reply:
React 18 and later, the createRoot function has been moved to a submodule called "react-dom/client". Your suggestion is highly appreciated, but due to the above reasons I will just have to import createRoot from react-dom/client.

 
![Screenshot from 2023-09-19 14-23-56](https://github.com/sagieramos/react-todo-app/assets/31627384/56971666-6381-4947-b530-d6280f90e422)
![Screenshot from 2023-09-19 14-33-56](https://github.com/sagieramos/react-todo-app/assets/31627384/5721f558-f17e-4273-9702-7f0a9c8ef8b3)

Reply:
@sagieramos, your work is great you done really well & thank you for bringing this into my knowledge.